### ISSUE TITLE
Typing play without any other commands causes a very slow response

### ISSUE THREAD

**Issue by Ehnryu** [aura-systems/Aura-Operating-System#224]
![image](https://user-images.githubusercontent.com/57133270/141673803-3567a9de-1ac4-4eb9-8abc-fae896062806.png)

When given no arguments play takes nearly a minute to finish

**Comment by valentinbreiz** [comment]
Hello @Ehnryu ,
It's normal that the "play" command takes some time because it is playing a melody with the PC Speaker. At each note Aura waits for a few milliseconds and then play the next one.
If you didn't ear anything, was your speaker enabled? Or which Virtual Machine are you using?

**Comment by Ehnryu** [comment]
I am using VMware under other as the wiki guide said but for some reason even with 100% volume it did not play

**Comment by RSTPlays** [comment]
Maybe the audio doesn't work on VMWare, I don't know because I've never used Console.Beep()

**Comment by Kiirx** [comment]
@RSTPlays it does for me

**Comment by RSTPlays** [comment]
Yeah, turns out there is. Maybe there’s audio issues form your side
